queue/group_remote_mongo_single.go

```diff
@@ -188,14 +184,22 @@ func (g *remoteMongoQueueGroupSingle) Get(ctx context.Context, id string, opts .
 		if err != nil {
 			return nil, errors.Wrap(err, "getting queue options")
 		}
-		queueOpts = mergeMongoDBQueueOptions(append([]MongoDBQueueOptions{g.opts.Queue}, mdbOpts...)...)
+
+		precedenceOrderedOpts := []MongoDBQueueOptions{g.opts.DefaultQueue}
```

Comment:
This block of code is reused, can we modulate it into a helper function?

Reply:
Done.